apps/dalmatiner_frontend/src/dalmatiner_dl_collection_h.erl

```diff
@@ -0,0 +1,41 @@
+-module(dalmatiner_dl_collection_h).
+-behaviour(cowboy_http_handler).
+
+-export([init/3, handle/2, terminate/3]).
+
+-ignore_xref([init/3, handle/2, terminate/3]).
+
+init(_Transport, Req, []) ->
+    {ok, Req, undefined}.
+
+
+-dialyzer({no_opaque, handle/2}).
+handle(Req, State) ->
+    Req0 = cowboy_req:set_resp_header(
+             <<"access-control-allow-origin">>, <<"*">>, Req),
+
+    {ContentType, Req1} = dalmatiner_idx_handler:content_type(Req0),
+    case ContentType of
+        html ->
+            F = fun (Socket, Transport) ->
+                        File = code:priv_dir(dalmatiner_frontend) ++
+                            "/static/collection.html",
+                        Transport:sendfile(Socket, File)
```

Comment:
Remove the dynamic function call on line 23. Only modules that define callbacks should make dynamic calls.